regolith/helpers/f_todohelper.py

```diff
@@ -63,47 +66,63 @@ def db_updater(self):
                     "Please set default_user_id in '~/.config/regolith/user.json', or you need to enter your group id "
                     "in the command line")
                 return
-        now = dt.date.today()
-        if not rc.end_date:
-            end_date = now
-        else:
-            end_date = date_parser.parse(rc.end_date).date()
+        person = document_by_value(all_docs_from_collection(rc.client, "people"), "_id", rc.assigned_to)
         filterid = {'_id': rc.assigned_to}
-        person = rc.client.find_one(rc.database, rc.coll, filterid)
         if not person:
             raise TypeError(f"Id {rc.assigned_to} can't be found in people collection")
-        todolist = person.get("todos",[])
-        if len(todolist) ==0:
+        todolist = person.get("todos", [])
+        if len(todolist) == 0:
             print(f"{rc.assigned_to} doesn't have todos in people collection.")
             return
-        index = 1
-        for todo in todolist:
-            if todo.get('status') not in ["finished"]:
-                todo["index"] = index
-                index+=1
-
+        now = dt.date.today()
         if not rc.index:
+            if not rc.certain_date:
+                today = now
+            else:
+                today = date_parser.parse(rc.certain_date).date()
+            for todo in todolist:
+                if not todo.get('importance'):
+                    todo['importance'] = 1
+                if type(todo["due_date"]) == str:
+                    todo["due_date"] = date_parser.parse(todo["due_date"]).date()
+                todo["days_to_due"] = (todo.get('due_date') - today).days
+                todo["order"] = todo['importance'] + 1 / (1 + math.exp(abs(todo["days_to_due"])))
+            todolist = sorted(todolist, key=lambda k: (-k['order'], k.get('duration', 10000)))
+            print("-" * 50)
             print("Please choose from one of the following to update:")
+            print("  (running_index) action (days to due date|importance|expected duration(mins))")
```

Comment:
in English we always put space between words and parens but no space inside parens, so (this is good) but(this is not good) and ( neither is is this ).

Can we put this informational line above the    ``print("-" * 50)`` line.  My eyes have found this confusing having it mixed iwth the actual data.

We may want another informational print statement above this that says, "if the list numbers are far from being in numerical order, please reorder them by running ????"